datafusion/src/physical_plan/expressions/mod.rs

```diff
@@ -84,9 +86,13 @@ pub use nth_value::NthValue;
 pub use nullif::{nullif_func, SUPPORTED_NULLIF_TYPES};
 pub use rank::{dense_rank, percent_rank, rank};
 pub use row_number::RowNumber;
+pub(crate) use stddev::is_stddev_support_arg_type;
+pub use stddev::{stddev_return_type, Stddev};
```

Comment:
Do we need to public `stddev_return_type` and `Stddev` as a lib function?

Reply:
👍